app/run-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
The little shell script that could! I love the improvements here. Thanks!

Can you please add a copyright header here w/ the both of us?